### PR TITLE
E0d/munin node

### PR DESCRIPTION
**Pull request by e0d**
@edx/devops role to install and configure munin-node

### CODE REVIEW

File: playbooks/roles/munin_node/meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (feanil) [inline playbooks/roles/munin_node/meta/main.yml]: nit: Do we need this meta file considering it's empty?

**Comment by feanil** [comment]
One nit but :+1: whether or not you decide to fix it.

File: playbooks/roles/munin_node/meta/main.yml (hunk 1 above)
Comment (e0d) [inline playbooks/roles/munin_node/meta/main.yml]: Was debating add common.

Comment (feanil) [inline playbooks/roles/munin_node/meta/main.yml]: I think this role has no need for common from what I saw so I don't see a need to add it unless we want to make this fully edx specific. I could go either way but can't personally think of a good reason to include common for this role.

**Pull request merged**